src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java

```diff
@@ -53,6 +53,9 @@
     @JsonProperty("HostsPath")
     private String hostsPath;
 
+    @JsonProperty("LogPath")
+    private String logPath;
```

Comment:
If it already exists in existing json samples, then extend deserialisation test case with check.